pull-requests/PullRequestTemplate.md

```diff
@@ -0,0 +1,44 @@
+Thank you for contributing.
+Before submitting this PR, please make sure:
+
+-------------------------------------
+* [ ] I have followed the guidelines in [Contributing as Developers](https://github.com/microsoft/code-with-engineering-playbook/blob/master/CONTRIBUTING.md).
+* [ ] I have added an explanation of what my changes do and why I would like you to include them.
```

Comment:
Would reword to "I have explained my changes and reasons why they should be included."